controllers/machineinventory_controller.go

```diff
@@ -55,13 +57,6 @@ func (r *MachineInventoryReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&elementalv1.MachineInventory{}).
 		Owns(&corev1.Secret{}).
-		Watches(
-			&source.Kind{
-				Type: &corev1.Secret{},
-			},
-			&handler.EnqueueRequestForOwner{
```

Comment:
ok to drop watching the secrets... a bit puzzled at the beginning about dropping the check on the child resources... but anyway, when we are in the Ready state we don't update the resources anymore.
Did we got to many calls from child objects?
Anyway, if I recall correctly, reconcile loops are called every 10/15 minutes.

Reply:
This was redundant, the  `Owns(&corev1.Secret{})` on the line above does exactly the same thing, this is even stated at the [documentation](https://github.com/kubernetes-sigs/controller-runtime/blob/v0.14.5/pkg/builder/controller.go#L107), I guess this were just leftovers of the time it was first coded.